common/src/main/java/org/apache/gravitino/dto/requests/PolicyCreateRequest.java

```diff
@@ -0,0 +1,170 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.gravitino.dto.requests;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonSubTypes;
+import com.fasterxml.jackson.annotation.JsonTypeInfo;
+import com.google.common.base.Preconditions;
+import java.util.Set;
+import javax.annotation.Nullable;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.ToString;
+import org.apache.commons.lang3.StringUtils;
+import org.apache.gravitino.MetadataObject;
+import org.apache.gravitino.dto.policy.PolicyContentDTO;
+import org.apache.gravitino.policy.Policy;
+import org.apache.gravitino.rest.RESTRequest;
+
+/** Represents a request to create a policy. */
+@Getter
+@EqualsAndHashCode
+@ToString
+public class PolicyCreateRequest implements RESTRequest {
+
+  @JsonProperty("name")
+  private final String name;
+
+  @JsonProperty("comment")
+  @Nullable
+  private final String comment;
+
+  @JsonProperty("policyType")
+  private final String policyType;
+
+  @JsonProperty(value = "enabled", defaultValue = "true")
+  private final Boolean enabled;
+
+  @JsonProperty("content")
+  @JsonTypeInfo(
+      use = JsonTypeInfo.Id.NAME,
+      include = JsonTypeInfo.As.EXTERNAL_PROPERTY,
+      property = "policyType",
+      defaultImpl = PolicyContentDTO.CustomContentDTO.class)
+  @JsonSubTypes({
+    // add mappings for built-in types here
+    // For example: @JsonSubTypes.Type(value = DataCompactionContent.class, name =
+    // "system_data_compaction")
+  })
```

Comment:
Can you please add the test to verify the polymorphism of the `PolicyContent`.

Reply:
Since there is no built-in type policy, we cannot test polymorphism now. However, I have tested it manually locally. Please check the test code.https://github.com/apache/gravitino/pull/7821/commits/116422790d4f99e341377a8b372448da3b4485db

Reply:
Also, we can keep those test, if we can support polymorphism, then we can update those tests.